<ui version="4.0" >
 <class>dlgtooltip</class>
 <widget class="QWidget" name="dlgtooltip" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QLabel" name="lbl_menu_top" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet" >
        <string notr="true" >border-radius: 0px;
border: none;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #90b8d7, stop: 1 #0c669d);
color: #ffffff;</string>
       </property>
       <property name="text" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">KBFX Tooltips Configuration&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="margin" >
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Tooltip: Text</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" >
          <property name="margin" >
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Turn your tooltip off or on.
You can see the current status of the tooltip to the right.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton" >
            <property name="text" >
             <string>Turn the tooltip 'ON'</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="minimumSize" >
           <size>
            <width>180</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>180</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet" >
           <string notr="true" >border-radius: 6px;
border: 2px solid #8f8f91;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f6f7fa, stop: 1 #dadbde);
font: 75 36pt;</string>
          </property>
          <property name="text" >
           <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-size:36pt; font-weight:600;">OFF&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin" >
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QLabel" name="lbl_menu_top_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet" >
        <string notr="true" >border-radius: 0px;
border: none;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #90b8d7, stop: 1 #0c669d);
color: #ffffff;</string>
       </property>
       <property name="text" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">KBFX Tooltips Configuration&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="margin" >
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Tooltip: Animation</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" >
          <property name="margin" >
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Turn your tooltip animation off or on.&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">You can see the current status of the tooltip to the right.&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2" >
            <property name="text" >
             <string>Turn the tooltip animation 'ON'</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="minimumSize" >
           <size>
            <width>180</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>180</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet" >
           <string notr="true" >border-radius: 6px;
border: 2px solid #8f8f91;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f6f7fa, stop: 1 #dadbde);
font: 75 36pt;</string>
          </property>
          <property name="text" >
           <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:36pt; font-weight:600;">ON&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin" >
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5" >
     <item>
      <widget class="QLabel" name="lbl_menu_top_3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet" >
        <string notr="true" >border-radius: 0px;
border: none;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #90b8d7, stop: 1 #0c669d);
color: #ffffff;</string>
       </property>
       <property name="text" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">KBFX Tooltips Configuration&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="margin" >
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Tooltip Text</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QLineEdit" name="lineEdit" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>KFBX Application Menu</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Type the text which you want to display in the tooltip of the KBFX kicker button.</string>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
